raiden/raiden_event_handler.py

```diff
@@ -185,6 +186,9 @@ def handle_send_secretrequest(
             raiden: 'RaidenService',
             secret_request_event: SendSecretRequest,
     ):
+        if reveal_secret_with_resolver(raiden, secret_request_event):
```

Comment:
Note: This is a first iteration and we are okay with the additional request since it requires user configuration, in the future we will probably optimize it away but it requires https://github.com/raiden-network/raiden/issues/1202

Reply:
OK.